src/github.com/couchbase/sync_gateway/rest/config.go

```diff
@@ -75,6 +82,7 @@ type ServerConfig struct {
 	Databases                      DbConfigMap     // Pre-configured databases, mapped by name
 	MaxHeartbeat                   uint64          // Max heartbeat value for _changes request (seconds)
 	ClusterConfig                  ClusterConfig   `json:"cluster_config"` // Bucket and other config related to CBGT
+	SkipRunmodeValidation          bool            // If this is true, skips any config validation regarding accel vs normal mode
```

Comment:
Actually it's only meant to be used on the command line, and stashing it in the ServerConfig was the easiest way to do.  I guess if it's easy we should expose this via json config too, so I'll add a json name.